app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/InfoPetFragment.java

```diff
@@ -0,0 +1,61 @@
+package org.pesmypetcare.mypetcare.activities.fragments;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.AutoCompleteTextView;
+import android.widget.Button;
+
+
+import com.google.android.material.datepicker.MaterialDatePicker;
+
+import org.pesmypetcare.mypetcare.R;
+import org.pesmypetcare.mypetcare.databinding.FragmentInfoPetBinding;
+
+import java.util.Objects;
+
+
+public class InfoPetFragment extends Fragment {
+    private FragmentInfoPetBinding binding;
+    private Button birthDate;
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        binding = FragmentInfoPetBinding.inflate(inflater, container, false);
+        setCalendarPicker();
+        setGenderDropdownMenu();
+        return binding.getRoot();
+    }
+
+
+    /**
+     * Configure the Dropdown menu of gender
+     */
+    private void setGenderDropdownMenu() {
+        AutoCompleteTextView gender = binding.inputGender;
+        ArrayAdapter<String> adapter = new ArrayAdapter<>(Objects.requireNonNull(getContext()),
+                R.layout.drop_down_menu_item, new String[] {getString(R.string.male), getString(R.string.female)});
+        gender.setAdapter(adapter);
+    }
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5213151)